assets/styles.css

```diff
@@ -0,0 +1,286 @@
+body {
+  overflow-x: hidden;
+}
+
+.body-wrapper {
+  width: 1210px;
+  margin: 0 auto;
+}
+
+.header-wrapper {
+  height: 100px;
+  color: white;
+  background-color: #f72210;
+}
+
+.font-head a {
+  font-size: 14px;
+}
+
+.logo-head {
+  color: #fff;
+  fill: white;
+  width: 400px;
+}
+
+.header-font {
+  font: bold 13px/24px "Roboto Condensed", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+}
+
+.rounded-pill {
+  background-color: #ffe400;
+  height: 30px;
+}
+
+.search-input {
+  height: 38px;
+  width: max-content;
+}
+
+.article-body {
+  padding-top: 10px;
+  margin-bottom: 15px;
+}
+
+.main-section {
+  padding: 25px;
+}
+
+.navbar-one a {
+  padding: 15px;
+}
+
+.navbar-one li {
+  border-right: 1px solid #dee2e6 !important;
+}
+
+.newsletter-signup {
+  max-width: 300px;
+  margin: 0 auto 30px;
+  border: 1px solid #e5e5e5;
+  background-color: #f4f5f6;
+  padding: 20px;
+}
+
+.icon-signup {
+  flex: none;
+  display: block;
+  width: 29px;
+  height: 28px;
+  background: url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain no-repeat #f72210;
+  margin-right: 10px;
+}
+
+.free-signup-text {
+  font: bold 19px/19px "Roboto Condensed", sans-serif;
+  text-transform: uppercase;
+}
+
+.login-text-icon {
+  display: block;
```

Comment:
- [ ] Some of the styles written directly in your CSS file can be achieved using Bootstrap classes.
Go through your stylesheets file again, carefully, and search in [the Bootstrap documentation ](https://getbootstrap.com/docs/4.0/getting-started/introduction/)the classes you can use to substitute some of that vanilla CSS. 
> That will come in handy to know when you need to quick style a project with Bootstrap in the future!
**Examples of styles that can be done with Bootsrap classes:**
- border none
- border radius
- font weight
- text align
- position
- position
- width 100%